e2e/js_image/runfiles.bzl

```diff
@@ -0,0 +1,129 @@
+"contains container helper functions for js_binary"
+
+load("@rules_pkg//:providers.bzl", "PackageFilegroupInfo", "PackageFilesInfo", "PackageSymlinkInfo")
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
```

Comment:
it's so beautiful that you can make this work purely by composing existing rules 🧙🏻